lib/atoms/h1/h1.component.js

```diff
@@ -1,15 +1,11 @@
-angular
-    .module('lnPatterns')
-    .component('lnAH1', lnAH1);
-
 var lnAH1 = {
-    restrict: 'E',
-    replace: true,
     templateUrl: 'lnPatterns/atoms/h1/template.html',
     bindings: {
-        title: '<',
-        class: '<'
+        title: '@',
```

Comment:
They are different things, and we may need to use them in different situations. < binds objects and @ binds constant strings. See section "Component-based application architecture" here: https://docs.angularjs.org/guide/component